src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -1834,6 +1843,9 @@
         - name: --os-sku
           type: string
           short-summary: The os-sku of the agent node pool.
+        - name: --ssh-access
+          type: string
+          short-summary: Update SSH setting for the node pool. Allowed values are ["disabled", "localuser"]. Use "disabled" to disable SSH access, "localuser" to enable SSH access using private key. Default value is "localuser".
```

Comment:
After defining the param with `arg_type=get_enum_type` and `default=CONST_SSH_ACCESS_LOCALUSER`, the `Allowed values are ["disabled", "localuser"].` and `Default value is "localuser".` could be removed.
```suggestion
          short-summary: Update SSH setting for the node pool. Use "disabled" to disable SSH access, "localuser" to enable SSH access using private key.
```